lib/src/Base/Stat/ProcessSampleImplementation.cxx

```diff
@@ -194,6 +204,32 @@ Sample ProcessSampleImplementation::computeSpatialMean() const
   return result;
 }
 
+/* Standard deviation accessor */
+Field ProcessSampleImplementation::computeStandardDeviation() const
+{
+  const UnsignedInteger size = getSize();
+  if (size == 0) return Field();
+  if (size == 1) return Field(mesh_, getDimension());
+  Sample meanValues(computeMean().getValues());
```

Comment:
this way you go twice through the data. You can compute the marginal variance incrementally then takes the square root of these values, in the spirit of the Monte Carlo update. It should be mush more efficient on large data sets.

Reply:
it's a tradeoff, because you also have to store the incremental mean estimate